pkg/node/node.go

```diff
@@ -2274,3 +2279,19 @@ func ExtractPort(urlString string) (string, error) {
 
 	return port, nil
 }
+
+func removeVolumePrefixFromName(volumeName string) string {
+	// Remove the tenant-prefix (Auth v2) from the volume name if exists so that the mount will not fail
+	extracted := ""
+	if strings.HasPrefix(volumeName, "csivol") {
```

Comment:
Instead of hardcoding 'csivol', it might be cleaner and more flexible to pull this value from an environment variable.